docs/nx-cloud/troubleshooting/ci-execution-failed.md

```diff
@@ -16,9 +16,9 @@ This behavior can be disabled by setting the [--require-explicit-completion](/ci
 
 ### A command was issued to stop all Nx Cloud agents
 
-Nx Cloud provides two commands to forcibly stop agents, [stop-all-agents and complete-ci-run](/ci/reference/nx-cloud-cli#npx-nxcloud-stopallagents).
+Nx Cloud provides a command to stop all agents: [npx nx-cloud complete-ci-run](/ci/reference/nx-cloud-cli#npx-nxcloud-completecirun).
```

Comment:
Since `stop-all-agents` does result in this behavior still, I think it should be mentioned here. We can make add something to point out that it's no longer the recommended command, but it will be confusing for someone that doesn't use `complete-ci-run` to come view the paragraph and see a totally separate command listed as the cause